napari/_qt/qt_main_window.py

```diff
@@ -656,6 +660,9 @@ def __init__(self, viewer: 'Viewer', *, show: bool = True) -> None:
         plugin_manager.discover_themes()
         self._setup_existing_themes()
 
+        # npe1 shim
```

Comment:
nit: can we be a bit more specific here and say "import and index all discovered shimmed npe1 plugins"

I'm particular about this because they're actually discovered elsewhere, but this separate step imports them.

Reply:
Done!